Sources/web3swift/Transaction/EIP1559Envelope.swift

```diff
@@ -147,9 +134,6 @@ extension EIP1559Envelope {
             self.to = ethAddr
         }
 
-        // MARK: workaround for gasPrice coming from nodes for EIP-1159 - this allows Oracle to work for now
```

Comment:
It's 1559 mistyped to 1159

Reply:
copy paste when I did the workaround... this PR is removing it